webextensions/_locales/fr/messages.json

```diff
@@ -133,250 +133,254 @@
   "tabDragBehaviorNotification_message_tree_bookmark": { "message": "créer des liens ou des marques-page à l'endroit où les onglets sont déposés" },
   "tabDragBehaviorNotification_message_tab_bookmark":  { "message": "créer un lien ou un marque-page à l'endroit où l'onglet est déposé" },
 
-  "warnOnCloseTabs_message":   { "message": "You are about to close $NUMBER$ tabs. Are you sure you want to continue?",
+  "warnOnCloseTabs_message":   { "message": "Vous êtes sur le point de fermer $NUMBER$ onglets. Voulez-vous vraiment continuer ?",
     "placeholders": {
       "NUMBER": { "content": "$1", "example": "10" }
     }},
-  "warnOnCloseTabs_warnAgain": { "message": "Warn me when I attempt to close multiple tabs" },
+  "warnOnCloseTabs_warnAgain": { "message": "Me prévenir quand j'essaie de fermer de multiples onglets" },
   "warnOnCloseTabs_close":     { "message": "Fermer les onglets" },
   "warnOnCloseTabs_cancel":    { "message": "Annuler" },
 
   "warnOnAutoGroupNewTabs_message":   { "message": "$NUMBER$ tabs are opened in a time. Do you want them to be grouped in a tree?",
     "placeholders": {
       "NUMBER": { "content": "$1", "example": "10" }
     }},
-  "warnOnAutoGroupNewTabs_warnAgain": { "message": "Demandez-moi quand plusieurs onglets sont ouvert en même temps" },
+  "warnOnAutoGroupNewTabs_warnAgain": { "message": "Demandez-moi quand plusieurs onglets sont ouverts en même temps" },
   "warnOnAutoGroupNewTabs_close":     { "message": "Grouper les onglets" },
   "warnOnAutoGroupNewTabs_cancel":    { "message": "Keep tabs flat" },
 
-  "bookmarkDialog_title":    { "message": "Titre:" },
-  "bookmarkDialog_url":      { "message": "Adresse:" },
-  "bookmarkDialog_parentId": { "message": "Dossier:" },
+  "bookmarkDialog_title":    { "message": "Titre :" },
+  "bookmarkDialog_url":      { "message": "Adresse :" },
+  "bookmarkDialog_parentId": { "message": "Dossier :" },
```

Comment:
If possible, add non breaking spaces instead of regular spaces

Reply:
To be honest I used no-break spaces but didn’t check existing strings. I think got them all in my local changes now.